src/tasks/tasks.ts

```diff
@@ -245,6 +254,14 @@ export class Tasks {
         if (!commitmentId) {
             throw new Error(ERRORS.CONTRACT.CAPTCHA_SOLUTION_COMMITMENT_DOES_NOT_EXIST.message)
         }
+
+        await this.dappUserCommit(
+            '5DPEcuBK23Hume855tKoyyKLGCHhjM9BJPHr6ebhbAQ5QE7G',
```

Comment:
not sure about the meaning of those hardcoded values?

Reply:
Just added for testing purpose. Removed now.